src/formatters/ExposureFormatter.cc

```diff
@@ -452,8 +449,8 @@ void ExposureFormatter<ImagePixelT, MaskPixelT, VariancePixelT>::delegateSeriali
     if (ip == 0) {
         throw LSST_EXCEPT(pex::exceptions::RuntimeError, "Serializing non-Exposure");
     }
-    std::shared_ptr<image::Wcs> wcs = ip->getWcs();
-    ar& * ip->getMetadata() & ip->_maskedImage& wcs;
+    std::shared_ptr<geom::SkyWcs const> wcs = ip->getWcs();
+    ar& * ip->getMetadata() & ip->_maskedImage;  // & wcs; // TODO: replace this with what?
```

Comment:
Why is the wcs removed here?  Do we no longer serialize it as before?

Reply:
SkyWcs is serialized in an entirely different way. Note that ExposureFormatter.cc is probably dead code.

Reply:
I could be wrong, but I think the butler still uses the formatter stuff.